js/apps/system/_admin/aardvark/APP/foxxes.js

```diff
@@ -90,13 +92,32 @@ installer.use(function (req, res, next) {
   options.setup = req.queryParams.setup;
   options.teardown = req.queryParams.teardown;
   let service;
+
+  let continueFoxxOperation = true;
+  const thisCoordinatorId = FoxxManager._getMyCoordinatorId();
+  if (thisCoordinatorId) {
+    if (req.body.coordinatorId) {
```

Comment:
What happens if you (maliciously) give a non-existent coordinator ID in this request?

Reply:
https://github.com/arangodb/arangodb/pull/18691/files#diff-970983882d6f4ad739c362b38dac0ef2d7cd1aa12cf57f024d31c74125c3f782R239-R246

Now being handled here.